src/HTTPBackend.js

```diff
@@ -62,14 +62,16 @@ export default function(options) {
 		delete headers.parameters;
 		delete headers.queryParameters;
 
-		return {
-			header: { ...headers, statusCode: 400 },
-			body: {
-				errors: {
-					message: err.message
-				}
+		/* Status code 412 means "precondition failed", which is sort of what happened
+		 * when the fetch api itself throws an Error.
+		 */
+		err.header = {...headers, statusCode: 412};
```

Comment:
This maintains backwards compatibility with middlewares (and request.js) that depend on doing `resp.header.statusCode` and `resp.header.correlationId`, etc.